### DISCUSSION TITLE
Anyone using Synology SurveillanceStation?

### DISCUSSION THREAD

**Discussion by kridan** [mrlt8/docker-wyze-bridge#556] [Q&A]
I'm interested in seeing how other people have configured their wyze cams with SurveillanceStation.

The only option when adding one (even as UserDefined and taking the RTP or HTTP stream) seems to *require* that a username/password be provided.

I know that the bridge is working perfectly (of course :) because I can open the streams just fine.  But the synology s/w seems to require the cam to have a username/password

What am I missing?

**Comment by mrlt8** [comment]
You could set a username and password for a stream or all streams in rtsp_simple_server using something like this:

```yaml

- RTSP_PATHS_ALL_READUSER=user
- RTSP_PATHS_ALL_READPASS=password

```

**Reply by Djie7** [reply]
Does this need to be added to the config of the Docker Wyze Bridge, edit in YAML?

**Reply by mrlt8** [reply]
yes, should go under `environment:` something like:

    environment:
        - WYZE_EMAIL=
        - WYZE_PASSWORD=
        - RTSP_PATHS_ALL_READUSER=user
        - RTSP_PATHS_ALL_READPASS=password

**Reply by KSPSlice** [reply]
Thanks. That worked for me as well. I have my Home Assistant set up as a VM in Synology, so I had to change to the IP of the VM rather than homeassistant.local. The Wyze Bridge is set up in Home Assistant, so just added the RTSP paths in the config in Home Assistant.

**Comment by kridan** [comment]
Thank you - that did the trick.  I really with these cams would be supported without having to stand up a wyzebridge container, or flash them all.

**Reply by mrlt8** [reply]
Many of the cameras actually have a kinesis webrtc server which is used for the web view and smart displays. We can pull some of the credentials with the ` - WEBRTC=True` env option, but still need to figure out a way to bridge the WebRTC-to-RTSP.
